src/screens/FeedScreen/Content.js

```diff
@@ -294,23 +292,38 @@ const Content = ({
             />
           </View>
         ) : null}
-        {images_url.length > 0 && (
-          <View style={[styles.containerImage]}>
+        {isBlurredPost && images_url.length > 0 && (
+          <View
+            style={[
+              {
+                height: hasComment ? dimen.normalizeDimen(315) : dimen.normalizeDimen(388)
+              }
+            ]}
+          />
+        )}
+        {!isBlurredPost && images_url.length > 0 && (
+          <View style={[styles.containerImage(isBlurredPost)]}>
             <ImageLayouter
               isFeed={true}
               images={images_url}
               onimageclick={() => onPress(showSeeMore)}
             />
           </View>
         )}
-      </BlurredLayer>
 
-      <TopicsChip
-        onLayout={calculateLineTopicChip}
-        topics={topics}
-        fontSize={normalizeFontSizeByWidth(14)}
-        text={message}
-      />
+        {!isBlurredPost && (
+          <TopicsChip
+            onLayout={calculateLineTopicChip}
+            topics={topics}
+            fontSize={normalizeFontSizeByWidth(14)}
+            text={message}
+            topicContainer={{
+              position: item.post_type === POST_TYPE_STANDARD ? 'absolute' : 'relative',
+              bottom: 0
+            }}
+          />
+        )}
+      </BlurredLayer>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The rendering of images and the `TopicsChip` component is now conditional based on whether the post is blurred. This seems like a good way to handle blurred posts differently, but again, make sure this doesn't introduce any unexpected behavior or visual issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->